folium/features.py

```diff
@@ -227,18 +229,20 @@ def __init__(self, data, style_function=None):
         super(GeoJson, self).__init__()
         self._name = 'GeoJson'
         if 'read' in dir(data):
```

Comment:
I am not opposed to this pattern: `read in dir(x)`, but I think `hasattr(object, name)` is more common?  Any special reason for doing that?